src/handlebars/releases.handlebars

```diff
@@ -82,12 +82,18 @@
                             <div class='small-dl-text'>\{{installer_extension}}</div>
```

Comment:
Let's choose active verbs, i.e. 'Install' and 'Download' (rather than 'Installer' and 'Download').

Reply:
done